### PR TITLE
fix links in blog

### PR DESCRIPTION
**Pull request by hunshcn**
find by using regex `(?<!])\[[^\]]*\]\((?!https?://|mailto:|about:blank)[a-z0-9](?!.*png|.*svg|.*jpg)`

### CODE REVIEW

File: content/en/blog/_posts/2023-10-05-sig-architecture-conformance-spotlight.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ sub-project](https://github.com/kubernetes/community/blob/master/sig-architectur
 bit about yourself, your role and how you got involved in Kubernetes.
 
 **Riaan Kleinhans (RK)**: Hi! My name is Riaan Kleinhans and I live in
-South Africa. I am the Project manager for the [ii-Team](ii.nz) in New
+South Africa. I am the Project manager for the [ii-Team](https://ii.nz) in New
```
Comment (Gauravpadam) [inline content/en/blog/_posts/2023-10-05-sig-architecture-conformance-spotlight.md:29]: Though trivial, We can accept this change as a good practice

**Review by Gauravpadam — COMMENTED** [review]
Thanks

In general, we don't maintain blogs that are older than one year

File: content/en/blog/_posts/2023-10-05-sig-architecture-conformance-spotlight.md (hunk 1 above)
Comment (hunshcn) [reply content/en/blog/_posts/2023-10-05-sig-architecture-conformance-spotlight.md:29]: I see. Do I need to revert the changes to the old content?

Comment (Gauravpadam) [reply content/en/blog/_posts/2023-10-05-sig-architecture-conformance-spotlight.md:29]: Yes, I'd do that

Comment (hunshcn) [reply content/en/blog/_posts/2023-10-05-sig-architecture-conformance-spotlight.md:29]: done

**Comment by Gauravpadam** [comment]
An informal /approve from me

**Review by sftim — COMMENTED** [review]
/approve

**Pull request merged**